From d08be22cd529c3c3338de497078d0d2de0008c9d Mon Sep 17 00:00:00 2001
From: Gurpreet Ghai <gghai@codeaurora.org>
Date: Tue, 25 Sep 2018 19:12:40 +0530
Subject: [PATCH 26/37] BT Dev: Increase array size for codec selection in Dev
 Op

Increase array size to accomodate additional codec
in Developer Option codec list

 Conflicts:
	packages/SettingsLib/res/values/arrays.xml

CRs-Fixed: 2322634

Change-Id: I1de7d70e675bffed25b1016efbd1ea908b24d97d
Signed-off-by: Volodymyr Zhdanov <wight554@gmail.com>
---
 packages/SettingsLib/res/values/arrays.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/SettingsLib/res/values/arrays.xml b/packages/SettingsLib/res/values/arrays.xml
index 9aac66229a3..ab895e5d541 100644
--- a/packages/SettingsLib/res/values/arrays.xml
+++ b/packages/SettingsLib/res/values/arrays.xml
@@ -166,6 +166,7 @@
         <item>2</item>
         <item>3</item>
         <item>4</item>
+        <item>5</item>
     </string-array>
 
     <!-- Summaries for Bluetooth Audio Codec selection preference. [CHAR LIMIT=50]-->
-- 
2.28.0

